pylxd/client.py

```diff
@@ -374,6 +385,14 @@ def events(self, websocket_client=None):
 
         client = websocket_client(self.websocket_url)
         parsed = parse.urlparse(self.api.events._api_endpoint)
-        client.resource = parsed.path
+
```

Comment:
As the function signature is being modified, the docstring ought to be updated to current best practice and document the types of the params and return types.  Could I suggest:

```Python
    """
    ...
    :param websocket_client: Optional parameter can be specified for implementation-specific handling of events as they occur.
    :type websocket_client: ws4py.client import WebSocketBaseClient
    :param event_types: dictionary of event types to propagate
    :type event_types: Dict[EventType]
    :returns: instance of the websocket client
    :rtype: Option[_WebsocketClient(), :param:`websocket_client`]
    """